channels-rke2.yaml

```diff
@@ -1373,7 +1373,7 @@ releases:
     featureVersions: *featureVersions-v1
   - version: v1.25.13+rke2r1
     minChannelServerVersion: v2.7.2-alpha1
-    maxChannelServerVersion: v2.8.99
+    maxChannelServerVersion: v2.9.99
```

Comment:
looks like when we moved from v2.7 to v2.8 we kept the maxVersion of some previously released versions to v2.7.99: 
https://github.com/rancher/kontainer-driver-metadata/blob/8d81713847c3f609f7d5a6b821386979f0e0dcb5/channels-rke2.yaml#L1351-L1353
so just wanted to confirm if it is okay to update maxVersion for each k8s version from v2.8.99 to v2.9.99 ?

Reply:
On `release/v2.8` we _currently_ have `maxChannelServerVersion: v2.8.99` - https://github.com/rancher/kontainer-driver-metadata/blob/release-v2.8/channels-rke2.yaml#L1376 ([permalink](https://github.com/rancher/kontainer-driver-metadata/blob/6bb1ef31a8df294d8d0f5954a1a3a8fb8c74f050/channels-rke2.yaml#L1376)).

Not sure where the commit in your comment is from. Overall I think we definitely want to extend versions range.

Please also note that even though we're likely to drop support for some older K8s versions in 2.9.0 (like 1.25) it's still safer to have these versions available in KDM in case somehow there is existing downstream cluster with such version. Supported versions range is handled more "globally" I believe. @kinarashah , please correct me if I'm wrong.

Reply:
@vardhaman22 @mitulshah-suse we'll update max versions for older k8s versions later in another PR if needed. Thanks for the reminder. 

@snasovich I need to revisit the code for rke2/k3s, but yes that's kinda true for rke1 - minRancherVersion decides what k8s versions are current for the corresponding rancher version but deprecateRancherVersion is the one that finally prevents cluster create/updates and we haven't used that in a while.